src/Search.js

```diff
@@ -1,23 +1,39 @@
-import React from "react";
+import React, { useState } from "react";
+import SearchButton from "./SearchButton";
+
+const Search = props => {
+  // console.log(props.search);
+  const [searchInput, setSearchInput] = useState("");
+  //const [onSubmit,setOnSubmit] = useState()
+
+  const handleSearchSubmit = event => {
+    event.preventDefault();
+    props.search(searchInput);
+  };
```

Comment:
You could add `setSearchInput("")` at the end to clear the input when submitting the form (which is common practice)